programs/client/TestHint.h

```diff
@@ -19,6 +15,10 @@ namespace DB
 ///
 /// - "-- { clientError 20 }" -- in case of you are expecting client error.
 ///
+/// - "-- { serverError FUNCTION_THROW_IF_VALUE_IS_NON_ZERO }" -- by error name.
+///
+/// - "-- { clientError FUNCTION_THROW_IF_VALUE_IS_NON_ZERO }" -- by error name.
+///
```

Comment:
typo?
```suggestion
/// - "-- { serverError FUNCTION_THROW_IF_VALUE_IS_NON_ZERO }" -- by error name.
///
```

Reply:
No, the comment shows that clientError and serverError accepts error code by it's name (not only by the numeric value of an error).

Reply:
Oh, ok, I didn't notice difference at first